src/top/utop.el

```diff
@@ -842,7 +842,7 @@ If ADD-TO-HISTORY is t then the input will be added to history."
   "Go to the beginning of line or to the end of the prompt."
   (interactive)
   (with-current-buffer utop-buffer-name
-    (if (= (point-at-bol) utop-prompt-min)
```

Comment:
Is this deprecated or what?

Reply:
Yeah, I made this in a separate commit to shut up the CI.